src/components/AAForm/AAOrganizationForm.vue

```diff
@@ -0,0 +1,196 @@
+        <!-- <BFormGroup id="orgLogoGroup" label="Logo" label-for="form-organization-logo">
+          <b-form-file
+            ref="logo"
+            @input="update"
+            v-show="(orgLogo && !validateState('logo',true)) || !orgLogo"
+            v-model="form.organization.image"
+            id="form-organization-logo"
+            @change="showImage"
+            v-validate="'size:50'"
+            :state="validateState('logo',true)"
+            multiple
+            name="logo"/>
+
+          <field-error-message :error="errors.collect('logo')"/>
+        </BFormGroup> -->
+        <BContainer v-if="orgLogo && validateState('logo',true)">
+          <div class="row">
+            <div class="col-11">
+              <img :src="orgLogo" class="logo">
+            </div>
+            <div class="col-11">
+              <button v-on:click="deleteLogo" type="button" class="close" aria-label="Close">
+                <span aria-hidden="true">&times;</span>
+              </button>
+            </div>
+          </div>
+        </BContainer>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-lg-6">
+        <BFormGroup id="orgTypeGroup">
+          <label data-req="*" for="form.organization.type">Type  </label>
+          <SCBDSelect
+            @input="update"
+            type="OrganizationType"
+            id="form.organization.type"
+            v-model="form.organization.types"
+            multi
+            tag-view
+            v-validate="'required'"
+            :state="validateState('organization type',form.organization.types.length)"
+            name="organization type"
+          />
+
+          <field-error-message
+            :error="errors.collect('organization type')"
+            :state="validateState('organization type',form.organization.types.length)"
+          />
+        </BFormGroup>
+      </div>
+      <div class="col-lg-6">
+        <BFormGroup id="hqcGroup">
+
+          <label data-req="*" for="form.organization.country">Location  </label>
+          <SCBDSelect
+            @input="update"
+            type="Countries"
+            id="form.organization.country"
+            v-model="form.organization.country"
+            tag-view
+            v-validate="'required'"
+            :state="validateState('Headquarters Country')"
+            name="Headquarters Country"
+          />
+
+          <field-error-message
+            :error="errors.collect('Headquarters Country')"
+            :state="validateState('Headquarters Country')"
+          />
+        </BFormGroup>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+
+  import AAFormMixin from '../../modules/AAFormMixin' 
+  import SCBDSelect from './controls/SCBDSelect' 
+  import Links from './controls/Links'
+
+  export default {
+    name: 'AAOrganizationForm',
+    mixins: [AAFormMixin],
+    components: {  SCBDSelect, Links },
+    props: {
+      value: {
+        type: [Array, Object],
```

Comment:
should this be not Array or Object?

Reply:
that is what it means.  The list of types are any of the types provided.  The component can be used to select a single entry as an object or many as an array.